### DISCUSSION TITLE
Xch confirmed sent did arrived targeted wallet

### DISCUSSION THREAD

**Discussion by adewelo** [Chia-Network/chia-blockchain#17380] [Support]
How can I retreive xch confirmed sent and never gotten to targeted wallet?

**Comment by BrandtH22** [comment]
Hey @adewelo , we may need more information and can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Something to keep in mind is if one uses the clawback feature, the intended recipient will need a 2.x version of chia and will need to claim the funds for them to appear in the recipients wallet.

If this is not the case then I would start by resyncing both the sending and receiving wallet with the GUI settings / advanced / resync option. This will close the GUI and upon reopening will sync the wallet from scratch. 

If the funds still have not appeared in the recipient or senders wallet I would urge you to join our discord server so we can further troubleshoot (we can also troubleshoot here but keep in mind that will take a bit more time to fully troubleshoot).

**Reply by adewelo** [reply]
Issue resolved by CoinEx. Thanks for your help

**Comment by BrandtH22** [comment]
@adewelo thank you for the confirmation. I will close out this ticket but let us know if any other questions arise 👍
